changelog/6613.improvement.md

```diff
@@ -0,0 +1,2 @@
+Added telemetry reporting. Reporting will be opt-out. More information can 
```

Comment:
I think the telemetry page is very verbose and since this doesn't require any change in user behaviour (e.g. like a change to our ML models). In addition to that users installing it will be notified again with a message in their CLI

Reply:
> The changelog is going to be pretty packaged, so I'd keep this one short

But should it really be _this_ changelog which is kept short then? Communities are usually very wary about these kind of changes and I'd like to avoid the impression that we're trying to slip anything in. Imo being fully transparent (and potentially too verbose) reduces risk.

Reply:
I don't think this is an issue, we have two documentation pages about this, including all the tracked events as well as a piece of information once you run this the first time. I really don't think users will feel that we are slipping this in. But I am also not sure debating this makes sense so I'll add two more sentences.